src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -207,6 +207,7 @@ private function translateAssets(array $translations, string $locale): void
         foreach ($translations as $id => $message) {
             $responses[$id] = $this->client->request('POST', sprintf('translations/%s/%s', rawurlencode($id), rawurlencode($locale)), [
                 'body' => $message,
+                'headers' => ['Content-Type' => 'text/plain'],
```

Comment:
This line should be added to all calls that have a string as body.

Reply:
I've changed only where I have tested.
Another place where you suggest to add is POST route to 'tags/%s.json'.
But I'm not sure if there 'text/plain' is suitable.
The documentation suggests 'application/json': https://localise.biz/api#!/tags/tagAssets
![obraz](https://user-images.githubusercontent.com/40465697/159802121-346d64c2-51ac-44b1-86b7-5cca1b6f1779.png)

Reply:
We should review the calls and put the content-type that make sense. Sending application/json only if we send actual json, etc. Right now we never do so. It'd be great confirming that the other endpoints work well of course.